src/main/java/bit/javaoop/AbstractManager.java

```diff
@@ -0,0 +1,53 @@
+package bit.javaoop;
+import java.util.List;
+
+public abstract class AbstractManager extends AbstractEmployee {
+
+    private final Float budget = 100_000f;
```

Comment:
`AbstractManager` should not have `budget` - only the `FixedBudgedManager` should have it.

Reply:
Besides, it should not be hardcoded as `100_000f` - instead it could be a constructor parameter :)